lib/ransack/context.rb

```diff
@@ -135,7 +135,7 @@ def ransackable_association?(str, klass)
     end
 
     def ransackable_scope?(str, klass)
-      klass.ransackable_scopes(auth_object).any? { |s| s.to_s == str }
+      klass.ransackable_scopes(auth_object).any? { |s| s.to_sym == str.to_sym }
```

Comment:
Depending on the ruby version, this could cause a memory leak from different string inputs being added to the symbol table.